Keas.Mvc/ClientApp/components/Spaces/SpacesTable.tsx

```diff
@@ -82,20 +82,29 @@ export default class SpacesTable extends React.Component<IProps, {}> {
                         className: "table-10p",
                         filterable: false,
                         Cell: row => (
-                            <span><i className="fas fa-hdd"></i> {row.original.equipmentCount}</span>
+                            <span><i className="fas fa-hdd"/> {row.original.equipmentCount}</span>
                         ),
                     },
                     {
                         Header: "Workstations",
                         headerClassName: "table-10p",
                         className: "table-10p",
-                        accessor: "workstationsCount",
+                        id: "workstationsCount",
+                        accessor: spaceInfo => {
+                            return [spaceInfo.workstationsInUse,spaceInfo.workstationsTotal];
```

Comment:
I think this would be a little more discoverable as an object instead of an array, since I find it hard to reason about what row.workstationsCount[1] is representing